certbot-ci/certbot_integration_tests/certbot_tests/test_main.py

```diff
@@ -1,10 +1,14 @@
 """Module executing integration tests against certbot core."""
 from __future__ import print_function
+import os
+import shutil
+from os.path import join
 
 import pytest
 from certbot_integration_tests.certbot_tests import context as certbot_context
 from certbot_integration_tests.certbot_tests.assertions import (
-    assert_hook_execution, assert_save_renew_hook
+    assert_hook_execution, assert_save_renew_hook, assert_certs_count_for_lineage,
```

Comment:
Still not sure I understand why importing `*` won't work here.

Reply:
I did not say that it would not work. I mean to say that if a module is designed to be used with an `import *`, then it should control what it leaks to the caller module by defining `__all__`. I think it is a bad practice to let a module leak for instance its own imported module in the caller module.

Now, I think also that it was over-engineering for a module that contains only 6 methods to design it with `import *`. I prefer the explicit approach consisting in deep importing the members we need from the module.

Reply:
I haven't seen the pattern of explicitly defining `__all__` before, could you point me to some resources explaining the benefits of doing that? Doesn't seem like it's too much of a problem to `import *` if we plan to use all/most of the assertions inside.

Reply:
I'm good with explicitly importing assertions we use here.

Reply:
Acknowledged.